internal/services/datafactory/data_factory.go

```diff
@@ -97,6 +97,18 @@ func flattenDataFactoryParameters(input map[string]*datafactory.ParameterSpecifi
 	return output
 }
 
+func expandDataFactoryAnnotations(input *[]string) *[]interface{} {
```

Comment:
You don't need this as there is one function called `utils.ExpandStringSlice` does the same thing.

Reply:
Nice! Fixed in 1179a4200a